service/controller/cluster.go

```diff
@@ -29,7 +29,11 @@ type ClusterFrameworkConfig struct {
 	TemplateVersion string
 }
 
-func NewClusterFramework(config ClusterFrameworkConfig) (*framework.Framework, error) {
+type Cluster struct {
+	*framework.Framework
+}
```

Comment:
This is just a basic type embedding `*framework.Framework`, future `*controller.Controller`. This embedding is to avoid imports and aliased imports in `service.go`. This could be `framework *framework.Framework`, but I don't see strong reason to do so since it isn't a library.